src/server.ts

```diff
@@ -567,6 +567,16 @@ export class MultisigServer {
       return
     }
 
```

Comment:
Aren't you in an error state if you are > maxSigners?

Reply:
The server should never get into that state if we use `=== maxSigners`, but I think it should still reject new clients in that state

Reply:
Changed to use `===` -- I think it's clearer for the logic not to consider a state that shouldn't logically be possible to reach